### PR TITLE
Feature revamp labs

### PR DESCRIPTION
**Pull request by Ashwin-op**

### CODE REVIEW

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss [hunk 1]
```diff
@@ -1,27 +1,46 @@
...
   }
 }
 
-.image{
-
-    @media screen and (min-width: 1300px){
-      width: 70vw;
-
+.image {
+  @media screen and (min-width: 1300px) {
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss]: Let's use the variables here

Comment (arpansac) [reply projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss]: We'll use 1366px

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss [hunk 2]
```diff
@@ -1,27 +1,46 @@
...
+    width: 70vw;
   }
-    @media screen and (max-width: 720px){
-      width: 80vw;
 
+  @media screen and (max-width: 720px) {
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss]: Closest variable width to be used

**Review by arpansac — CHANGES_REQUESTED** [review]
Just some minor changes requested :)

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss (hunk 1 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss]: Done

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss (hunk 2 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss]: Done

**Pull request merged**